fe/fe-core/src/main/java/com/starrocks/planner/FileScanNode.java

```diff
@@ -503,7 +505,9 @@ private void getFileStatusAndCalcInstance() throws UserException {
         Preconditions.checkState(fileStatusesList.size() == fileGroups.size());
 
         if (isLoad() && filesAdded == 0) {
-            throw new UserException("No source file in this table(" + targetTable.getName() + ").");
+            List<String> filePaths =
+                    fileGroups.stream().map(BrokerFileGroup::getFilePaths).flatMap(List::stream).collect(Collectors.toList());
+            ErrorReport.reportAnalysisException(ErrorCode.ERR_NO_FILES_FOUND, String.join(", ", filePaths));
```

Comment:
may want to cut off the filepaths join, if there are too many filepaths.

Reply:
ok